### PR TITLE
create pipeline

### PR DESCRIPTION
**Pull request by jpoo90**
Basic Karma Pipeline to run tests. 
- Tests for this pipeline are pending.

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
ok to test

File: README.md [hunk 1]
```diff
@@ -1 +1,89 @@
...
+# Overview
+
+Gulp Pipeline that uses Karma to run tests.
+
+_repo_: `ssh://git@github.com:kenzanmedia/pipeline-test-karma.git`
+
+_jenkins_: `https://kenzan.ci.cloudbees.com/job/CI-pipelines-test-karma/`
+
+## Install
+`npm install git+ssh://git@github.com/kenzanmedia/pipeline-test-karma.git`
```
Comment (obuckley-kenzan) [inline README.md]: we should change this to follow standard install conventions
`npm install pipeline-test-karma`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    "karma-chrome-launcher": "0.2.1",
+    "karma-coverage": "0.5.2",
+    "karma-jasmine": "0.3.6",
+    "karma-mocha": "0.2.0",
+    "karma-mocha-reporter": "1.1.1",
+    "karma-phantomjs-launcher": "0.2.1",
+    "lazypipe": "1.0.1",
+    "mocha": "2.3.3",
+    "phantomjs": "1.9.18",
+    "pipeline-handyman": "git+ssh://git@github.com/kenzanmedia/pipeline-handyman.git#aeb41b4423ca07d5bab17ce28e6bdfb1b4e7b364",
```
Comment (obuckley-kenzan) [inline package.json]: all the keystone pipelines can be updated and installed over NPM now.

**Comment by obuckley-kenzan** [comment]
this will continue to fail until package.json is updated to reflect that keystone repos have moved from _kenzanmedia_ to _kenzanlabs_

**Comment by paul-barry-kenzan** [comment]
@thescientist13 
This PR has been dangling for a while now, and since I'm making updates, I think we can close this. Thoughts?

**Pull request closed without merging**